cfme/infrastructure/virtual_machines.py

```diff
@@ -231,7 +231,7 @@ class TemplatesToolbar(InfraGenericDetailsToolbar):
     @property
     def is_displayed(self):
         expected_name = self.context['object'].name
-        expected_provider = self.context['object'].provider.name
+        expected_provider = self.context['object'].provider
```

Comment:
`expected_provider` is not a string anymore here? There is a comparison below `relationship_provider_name == expected_provider`. Does it work with that change?

Reply:
Yes it works, expected_provider is a string, I check it on miq shell and pdb it gives the desired output i.e provider name.
@quarckster

Reply:
@quarckster I agree with you here, this method isn't working. the provider will be an object, not a string.

Navigating to vm details and calling view.is_displayed returns False, this is broken.

```
In [19]: view.context['object'].provider
Out[19]: <cfme.infrastructure.provider.rhevm.RHEVMProvider name='rhv41', key='rhv41', zone='default'>

Reply:
REQUIRED: @anurag03, please double check your change. provider is object

Reply:
Yes I made a mistake in initializing the Vm object that caused the confusion. Correctly said will revert the changes.
@izapolsk @mshriver @quarckster